src/daemon/org.projectatomic.rpmostree.xml

```diff
@@ -0,0 +1,195 @@
+<!DOCTYPE node PUBLIC
+"-//freedesktop//DTD D-BUS Object Introspection 1.0//EN"
+"http://www.freedesktop.org/standards/dbus/1.0/introspect.dtd">
+<node name="/" xmlns:doc="http://www.freedesktop.org/dbus/1.0/doc.dtd">
+
+  <interface name="org.projectatomic.rpmostree.Manager">
+    <property name="DefaultSysroot" type="o" access="read"/>
+
+    <!--
+         Returns a object path for the Sysroot
+         at the give path. If the path is not
+         not a valid sysroot an error is returned.
+    -->
+    <method name="GetSysroot">
+      <arg name="path" type="s" direction="in"/>
+
+      <arg name="sysroot" type="o" direction="out"/>
+    </method>
```

Comment:
I don't recall if this was discussed in our meeting, but this method and the allowance for multiple `Sysroot` interfaces seems like overkill to me at this point.  Is there a use case for this method now?